images/arch/Containerfile

```diff
@@ -23,3 +23,6 @@ RUN yes | pacman -Scc
 
 # Enable sudo permission for wheel users
 RUN echo "%wheel ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/toolbox
+
+# Enable the use of p11-kit-client.so to access CA certificates from the host
+RUN mkdir --parents /etc/pkcs11/modules
```

Comment:
@Foxboron do you think the `libp11-kit` package should also create the `/etc/pkcs11/modules` directory?  It already creates `/usr/share/p11-kit/modules`, which isn't quite the same thing, but similar and related to the PKCS#11 modules that it ships.

Currently, the `p11-kit` package creates `/etc/pkcs11/pkcs11.conf.example`, and I wonder if it should be moved to the `libp11-kit` package.  I think the global configuration file (see `pkcs11.conf(5)`) affects both the libraries and executables.

Reply:
I checked with the package maintainer and he doesn't have a strong opinion. I'll take a look at the packaging when I have time and see if we can harmonize this.

Reply:
Thanks for following up!  :)